integration/docker/Dockerfile

```diff
@@ -42,6 +42,12 @@ ARG ENABLE_DYNAMIC_USER=false
 RUN apk --no-cache --update add bash libc6-compat shadow tini && \
```

Comment:
How about changing to

```bash
RUN apk --no-cache --update add bash libc6-compat shadow tini libbz2=1.0.6-r7 sqlite-libs=3.28.0-r3 && \
       rm -rf /var/cache/apk/*
```

Reply:
agree that these additions should be added into this line

instead of pinning down the desired version, can we specify these as minimum versions?
`RUN apk --no-cache --update add bash libc6-compat shadow tini 'libbz2>=1.0.6-r7' 'sqlite-libs>=3.28.0-r3' && \`

Reply:
ok